wordpress-mobile/WordPress-Android
# Increase Cursor Window Size to load larger posts

This [fix has been in Debug builds for over 2 years now](https://github.com/wordpress-mobile/WordPress-Android/pull/12582), and also [in use on Woo](https://github.com/woocommerce/woocommerce-android/blob/trunk/WooCommerce/src/main/kotlin/com/woocommerce/android/WooWellSqlConfig.kt#L73).  It is time to enable it for Release on WP/JP too to avoid crashes due to SQLiteBlobTooBigException.  We could also look at increasing size to 20MB or more later.

Fixes #18727 

> **Warning**
>
- [x] First Merge this [fix in FluxC](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2776) 
- [x] Publish a new FluxC version
- [x] Next update the new FluxC version here
- [x] Then merge this PR

To test:

Smoke test the app with really large posts. 

1. Launch WP/JP app release build (You can use AS to build a release version)

- Generate 100kb of text and save it into the clipboard (eg on https://www.lipsum.com/)
- Open a new post on the web
- Paste the text 50 times (50x 100kb)
- Publish the post
- Go to Post List in the app and notice the app crashes

2. Use this build in release version, and 
3. test it again with the same post, with reduced content size ~4MB
4. Verify that crash doesn't occur

> **Note**
> If the crashes are still occurring, we may need to increase it to 20MB



| Before | After |
|--------|--------|
| <img src="https://user-images.githubusercontent.com/1509205/112110512-ffe1e080-8b80-11eb-8394-55b6de766624.gif" width="320"></kbd>    |     <kbd><img src="https://user-images.githubusercontent.com/1509205/112110526-053f2b00-8b81-11eb-8d84-71ab8c48075c.gif" width="320">|


## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Smoke tested

3. What automated tests I added (or what prevented me from doing so)
Existing tests


PR submission checklist:
